### ISSUE TITLE
Foundry Server Debug Log shows warning... 

### ISSUE THREAD

**Issue by karpana** [League-of-Foundry-Developers/foundryvtt-forien-quest-log#11]
When I restart my Foundry instance, this message is continually present in the `debug.log` file.

Foundry Version: v9 build 249
System: PF1
Host/Server Operating System: Raspberry Pi 4.

```
{"level":"warn","message":"Package forien-quest-log contains 2 compendium definitions that still use the deprecated 'entity' field instead of 'type'.","timestamp":"2022-02-07 09:18:04"}
```

**Comment by typhonrt** [comment]
Thanks @karpana for the heads up; this isn't a breaking issue on v9, but indeed is a change in v9. `entity` changed to `type` in the manifest for packs in `module.json`.  There probably will be one more release of FQL before the new quest log module comes out and I'll update things.

**Comment by karpana** [comment]
Definitely not a breaking issue; module seems to work just fine. (which, by the way, my players are LOVING!!!)

But I do suspect this warning is being thrown by Foundy Core itself to "advise" of a deprecation situation that could be a potential future breaking change.

**Comment by karpana** [comment]
Looking forward to it @typhonrt 
(Will continue lurking on your discord for news or a beta test...)

**Comment by typhonrt** [comment]
Good news folks... This is fixed pending release.  There will be a FQL 0.7.10 release just after Foundry core v10 reaches "stable". This release works on v9 & v10 and includes several other quality of life updates that will work on either Foundry version. 

I'll close this issue after FQL 0.7.10 releases soon.

**Comment by typhonrt** [comment]
Hi @karpana. This issue is now fixed in the public release of FQL 0.7.10 available for Foundry VTT v9 & v10.
